velox/functions/sparksql/StringToMap.h

```diff
@@ -23,80 +23,105 @@ namespace facebook::velox::functions::sparksql {
 template <typename T>
 struct StringToMapFunction {
   VELOX_DEFINE_FUNCTION_TYPES(T);
+  static StringView entryDelimiter_;
+  static StringView keyValueDelimiter_;
 
   // Results refer to strings in the first argument.
   static constexpr int32_t reuse_strings_from_arg = 0;
-
+  // One parameter
```

Comment:
nit: remove this comment as it can be easily known through the function definition.

Reply:
done